src/webgpu/api/operation/render_pipeline/sample_mask.spec.ts

```diff
@@ -311,7 +317,7 @@ class F extends GPUTest {
     passEncoder.setBindGroup(0, uniformBindGroup);
     passEncoder.setStencilReference(kStencilReferenceValue);
 
-    if (rasterizationMask === 15) {
+    if (rasterizationMask === 2 ** sampleCount - 1) {
```

Comment:
Possible to make this conditional on sampleCount==1 instead, and maybe make the quad smaller so it covers only the center? There's no need to optimize by using a single quad for the all-four-samples case, the code below will cover it fine.